src/localization.py

```diff
@@ -46,6 +46,8 @@
 # Setting home as per the confinement
 home = snap_real_home if 'SNAP' in os.environ else snap_home
 
+# Setting Desktop variable
+desktopenv = os.getenv("XDG_CURRENT_DESKTOP")
```

Comment:
```suggestion
```
Since this variable is only used in main_window.py, there is no reason why it should be in this file.